tools/pythonpkg/cibw.toml

```diff
@@ -4,7 +4,7 @@
 [tool.cibuildwheel]
 environment = "PIP_CONSTRAINT='build-constraints.txt'"
 before-build = 'pip install oldest-supported-numpy'
-before-test = 'pip install --prefer-binary "pandas>=0.24" pytest-timeout mypy "psutil>=5.9.0" "requests>=2.26" fsspec && (pip install --prefer-binary "pyarrow>=8.0" || true)'
+before-test = 'pip install --prefer-binary "pandas>=0.24" pytest-timeout mypy "psutil>=5.9.0" "requests>=2.26" fsspec   && (pip install --prefer-binary "pyarrow>=8.0" || true) && (pip install --prefer-binary "torch" || true) && (pip install --prefer-binary "polars[pyarrow]" || true)'
```

Comment:
Is it worth just moving this to just using the requirements file instead? Feels silly to duplicate the deps

Reply:
jup, agree

Reply:
The only issue I see with this is that not all these libraries have wheels or compile in all our Python/Ci machinery.
Any ideas how we can install from the requirement while skipping errors?
Here this is done by:
e.g., 
`(pip install --prefer-binary "polars[pyarrow]" || true)`